include/SFML/Graphics/VertexBuffer.hpp

```diff
@@ -338,6 +338,15 @@ class SFML_GRAPHICS_API VertexBuffer : public Drawable, private GlResource
     Usage         m_usage{Stream};                        //!< How this vertex buffer is to be used
 };
 
+////////////////////////////////////////////////////////////
+/// \brief Swap the contents of one vertex buffer with those of another
+///
+/// \param left First instance to swap
+/// \param right Second instance to swap
+///
+////////////////////////////////////////////////////////////
+void swap(VertexBuffer& left, VertexBuffer& right) noexcept;
```

Comment:
Why did you add this free function? Does it mean we should make `sf::VertexBuffer::swap` a private method that is merely an implementation detail of this overload?

Reply:
`std::is_nothrow_swappable_v` requires a non-member swap being available. There are people on both sides of the fence on whether to have only 1 or both implementations, but I personally don't see why we can't just keep both especially since removing the member function at this point would just be unnecessarily causing API breakage without much benefit.